app/mailers/mail_preview.rb

```diff
@@ -110,13 +110,21 @@ def account_deleted
   end
 
   def cinstance_plan_changed
-    plans     = ServicePlan.last(2)
-    cinstance = FakeContract.new(plans.first, plans.second, Account.providers.last, Service.last, Account.last, '1')
+    plans     = ApplicationPlan.last(2)
+    cinstance = build_fake_contract(*plans)
     event = Cinstances::CinstancePlanChangedEvent.create(cinstance, User.last)
 
     NotificationMailer.cinstance_plan_changed(event, receiver)
   end
 
+  def cinstance_plan_changed_for_buyer
```

Comment:
This method and the previous one have a lot in common, it is literally everything the same except the kind of Event that it creates and the method in NotificationMailer that calls, which also have a lot in common between them but not as much as these 2 🤔 
So maybe these 2 can be refactored 🤔

Reply:
Fixed. Thanks.